commit 9fec5350a0260dea619cfc4ac7bbb90003ef180b (refs/changes/30/130/2)
Author: Dave Pacheco <dap@joyent.com>
Date:   2016-07-22T00:53:30+00:00 (3 years, 3 months ago)
    
    TOOLS-1494 crrestore should pull latest images
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/bin/crrestore b/bin/crrestore
index a933acf..9a24958 100755
--- a/bin/crrestore
+++ b/bin/crrestore
@@ -118,6 +118,12 @@ function main
 	gr_backup_path="$1"
 	gr_configure || fail "failed to configure"
 
+	echo "Pulling latest copies of cr.joyent.us images ... "
+	if ! docker pull $gr_image_appserver ||
+	   ! docker pull $gr_image_frontdoor; then
+	   	fail "failed"
+	fi
+
 	echo -n "Creating database volume container ... "
 	docker run --name=$gr_name_volume_db \
 	    -v /gerrit-postgres-db-data $gr_image_volumes \
